### DISCUSSION TITLE
"Variables size limit. The maximum allowed variables size is 50Kb" after upgrading subscription

### DISCUSSION THREAD

**Discussion by SoVeryAwake** [emailjs-com/help#132] [Q&A]
Hello!

I upgraded to the $50 tier to send large images, but I still receive this error. I can't tell if maybe I just haven't been properly provisioned yet, or if maybe I'm using the attachment variable incorrectly. 

Any guidance is appreciated :)

**Comment by xr0master** [comment]
Hi,

The error with 50Kb means that the attachment you send is not recognized as an Attachment. This can happen if the dynamic variable for your attachment does not match the variable in your template.

Please refer to the information below
https://www.emailjs.com/docs/user-guide/file-attachments/
